yoga/Yoga.c

```diff
@@ -2671,8 +2669,33 @@ static void YGNodelayoutImpl(const YGNodeRef node,
               case YGAlignStretch: {
                 child->layout.position[pos[crossAxis]] =
                     currentLead + YGNodeLeadingMargin(child, crossAxis, availableInnerWidth);
-                // TODO(prenaux): Correctly set the height of items with indefinite
-                //                (auto) crossAxis dimension.
+
+                // remeasure child with the line height as it as been only measure with the
+                // parents height yet
+                if (child->style.dimensions[dim[crossAxis]].unit == YGUnitUndefined) {
```

Comment:
make use of YGNodeIsStyleDimDefined here (it is getting some additions in a diff coming soon)